contrib/pg_tde/expected/no_provider_error.out

```diff
@@ -1,7 +1,7 @@
 CREATE EXTENSION pg_tde;
 -- should fail
 CREATE TABLE t1 (n INT) USING tde_heap;
-ERROR:  failed to retrieve principal key. Create one using pg_tde_set_principal_key before using encrypted tables.
+ERROR:  failed to retrieve the key. Create one using pg_tde_set_key before using encrypted tables.
```

Comment:
Don't think there should be a "the" here, but neither of us is a native speaker.

Reply:
@nastena1606 It should be `a` I guess, right?